cli/cmd/prometheus/port_forward.go

```diff
@@ -31,16 +31,24 @@ func prometheusPortForward(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not port-forward Prometheus: %w", err)
 	}
 
+	if err := PortForwardPrometheus(port); err != nil {
+		return err
+	}
+
+	select {}
+}
+
+func PortForwardPrometheus(listenPort int) error {
 	k8sClient := k8s.NewClient()
```

Comment:
Why do we need to create k8s client every time? Cannot we create it once and pass it around?